microshift_networking/microshift-configuring-routes.adoc

```diff
@@ -0,0 +1,52 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-configuring-routes"]
+= Configuring routes
+include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
+:context: microshift-configuring-routes
+
+toc::[]
+
+You can configure routes in {microshift-short} for networked clusters.
+//Q: assume this is not applicable to anything that is disconnected or offline? what would be a microshift use case for configuring routes?
+
+//OCP module, edit with care; Creating an insecure/http route
+include::modules/nw-creating-a-route.adoc[leveloffset=+1]
+
+//Q: HTTP Strict Transport Security routes use projects; can MicroShift users still use these configs without logging in to a project? https://docs.openshift.com/container-platform/4.15/networking/routes/route-configuration.html#nw-enabling-hsts_route-configuration
```

Comment:
Yes, MicroShift equivalent would be namespaces, which are also included in almost every command in that document.